chore: add contributor spotlight markdown file (#264)
mdn/mdn: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md

> # Nominating [GitHub username] as a [role]

Reviewer: Should we name this file `index.md`? Also we can follow lower case for the folder name. How about naming the folder something along these lines:
`.github/nomination_pull_request_template/index.md`

Author: From what I can tell from the GitHub docs, pull request templates are still very primitive :) and so, you can only have one, really. They have some guidance as to naming, and this is one of the ways. It is also how we do it on other repos. On Yari with have it at the root: https://github.com/mdn/yari/tree/main/.github

> Thank you for taking the time to nominate a fellow contributor. Make sure you read our [list of prerequisites](https://developer.mozilla.org/en-US/docs/MDN/Community/Users_teams) for the relevant role before submitting a nomination.

Reviewer: ```suggestion
Thank you for taking the time to nominate a fellow contributor. Make sure you read our [list of prerequisites](https://developer.mozilla.org/en-US/docs/MDN/Community/Users_teams) for the relevant role before submitting a nomination.
Please submit information for the person you are nominating in the sections listed below.
```

> ## GitHub username (required)

Reviewer: ```suggestion
## GitHub username of the nominee (required)
```

> ## GitHub profile link (required)

Reviewer: ```suggestion
## GitHub profile link of the nominee (required)
```

> ## Why are you nominating this person? (required)

Reviewer: ```suggestion
## Reason for nominating this person (required)
```

Tell us why you are nominating this person to become a spotlight contributor.

> ## Supporting evidence (required)

Reviewer: ```suggestion
## Supporting evidence for nominating this person (required)
```

> Provide evidence to support your nomination such as contributions and interactions with the community.
> Links to [commit logs](https://github.com/mdn/yari/commits?author=schalkneethling) in a repository are also appropriate.

Reviewer: ```suggestion
Provide evidence to support your nomination. The supporting information can include a list of their contributions and interactions with the community.
You can also provide links to [commit logs](https://github.com/mdn/yari/commits?author=schalkneethling).
```
